components/screens/ContactScreen/ContactScreen.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { View, Text, ScrollView, Button } from 'react-native';
+import { Styles } from './ContactStyle'
```

Comment:
Missing an `s` in _Styles_ here. This causes the app to crash.
Once this is fix though this should be good to merge with `main`